contracts/svm/Cargo.toml

```diff
@@ -19,6 +19,6 @@ codegen-units = 1
 anchor-lang = ">=0.29.0"
 anchor-client = ">=0.29.0"
 anchor-spl = ">=0.29.0"
-litesvm = "0.1.0"
-solana-program-test = "1.17.3"
-solana-sdk = "1.17.3"
+litesvm = ">=0.1.0"
+solana-program-test = ">=1.17.3"
+solana-sdk = ">=1.17.3"
```

Comment:
leaving these loose so people with anchor 0.29.0 can still use the express-relay crate if they want